android/KMAPro/build.sh

```diff
@@ -1,14 +1,21 @@
 #!/bin/bash
 # Build KMAPro
 
-display_usage ( ) {
-    echo "build.sh [-no-daemon] [-debug]"
-    echo
-    echo "Build Keyman for Android"
-    echo "  -no-daemon              Don't start the Gradle daemon. Use for CI"
-    echo "  -debug                  Compile only Debug variant"
-    exit 1
-}
+# Set sensible script defaults:
+# set -e: Terminate script if a command returns an error
+#set -e
```

Comment:
I had to comment out `set -e` so if `curl` fails to download, we'd know about it and die with a message.  (ll 89 and 99)

Reply:
https://superuser.com/a/940514/521575 shows techniques to avoid doing this. We don't want to lose this safety, I think.